jobs/pr_gate/docker/monorail/config.json

```diff
@@ -2,7 +2,7 @@
     "CIDRNet": "172.31.128.0/22",
     "amqp": "amqp://localhost",
     "apiServerAddress": "172.31.128.1",
-    "apiServerPort": 9090,
+    "apiServerPort": 9030,
```

Comment:
the apiServerPort is the port that is used by the nodes to access the southbound interface.  Changing this to 9030 will not make the nodes communicate directly with on-taskgraph via port 9030 instead of the on-http 9080 southbound interface.